mandatory/2-function-creation.js

```diff
@@ -15,17 +20,31 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  let conditions = typeof num === "number" && num % 2 === 0 && num <= 100;
+  // The below code is the longer solution
```

Comment:
I wouldn't suggest a longer solution since the condition is pretty straightforward. We would even do ``return num % 2 === 0 && num <= 100;`` without saving it into a variable. It is desirable if the condition is not complex :)

Reply:
Ahh! got it now. No need to check the data type of the number.